logger/logger.go

```diff
@@ -54,15 +54,27 @@ func SetLogger(zl *zap.Logger) {
 	logger = &Logger{zl.Sugar()}
 }
 
+const JSONLogFileName = "log.jsonl"
```

Comment:
As mentioned in Code Climate, let's make this private rather than document a public variable that's only used internally.

Reply:
Thanks. Fixed in 305e9e6